scraper/pandalab.py

```diff
@@ -0,0 +1,95 @@
+import httpx
+from httpx import TimeoutException
+import json
+import logging
+import urllib.parse as urlparse
+
+from datetime import datetime, timedelta
+from pytz import timezone
+from urllib.parse import parse_qs
+
+timeout = httpx.Timeout(30.0, connect=30.0)
+session = httpx.Client(timeout=timeout)
+logger = logging.getLogger('scraper')
+
+geo = {}
+items = {}
+insee = {}
+
+def getSchedules(pharmacyId, practitionerId, reasonId):
+    base_url = f"https://diapatient-api.diatelic.net/public/v1/appointment/schedule/{pharmacyId}"
+    payload = {"serviceProvider": "ICT", 
+               "practitionerId": practitionerId, 
+               "reasonId": reasonId}
+    try:
+        headers = {'Content-type': 'application/json', 'Accept': '*/*'}
+        r = session.post(base_url, data=json.dumps(payload), headers=headers)
+        r.raise_for_status()
+        return r.json()
+    except httpx.HTTPStatusError as hex:
+        return []
+
+def getReasons(pharmacyId):
+    base_url = f"https://diapatient-api.diatelic.net/public/v1/appointment/ict/practitioner/{pharmacyId}/reasons"
+    try:
+        r = session.get(base_url)
+        r.raise_for_status()
+        return(r.json())
+    except httpx.HTTPStatusError as hex:
+        return []
+
+def getPractitionerId(finessGeo):
+    base_url = f"https://diapatient-api.diatelic.net/public/v1/appointment/ict/pharmacy/{finessGeo}"
+    r = session.get(base_url)
+    r.raise_for_status()
+    return r.json()
+
+def cp_to_insee(cp):
```

Comment:
What about moving this in utils?

Reply:
je ferai la modif quand la PR#95 sera passée

Reply:
voilà maintenant j'importe depuis departements.py